### DISCUSSION TITLE
Preset or example templates

### DISCUSSION THREAD

**Discussion by smrbu** [ProxeusApp/community#7] [Ideas]
To onboard a new user it could be very helpful to have already setup samples in the admin panel under workflows, templates and forms. Kind of like what you find in the google docs template gallery. Such samples not only allow new users to get inspired, but also play around and understand the applications in real time.

**Comment by loleg** [comment]
Note that Proxeus already has [demo content](https://github.com/ProxeusApp/proxeus-core/tree/master/demo) which can be optionally deployed with a new server.
